upgrade/upgrade.go

```diff
@@ -219,23 +219,23 @@ func doResourceUpgrade(namespace string, lhClient *lhclientset.Clientset, kubeCl
 		return err
 	}
 
-	// When lhVersionBeforeUpgrade < v1.5.0, it is v1.4.x. The `CheckUpgradePathSupported` method would have failed us out earlier if it was not v1.4.x.
+	// When lhVersionBeforeUpgrade < v1.6.0, it is v1.5.x. The `CheckUpgradePathSupported` method would have failed us out earlier if it was not v1.5.x.
 	resourceMaps := map[string]interface{}{}
-	if semver.Compare(lhVersionBeforeUpgrade, "v1.5.0") < 0 {
-		logrus.Info("Walking through the resource upgrade path v1.4.x to v1.5.0")
-		if err := v14xto150.UpgradeResources(namespace, lhClient, kubeClient, resourceMaps); err != nil {
+	if semver.Compare(lhVersionBeforeUpgrade, "v1.6.0") < 0 {
+		logrus.Info("Walking through the resource upgrade path v1.5.x to v1.6.0")
+		if err := v15xto160.UpgradeResources(namespace, lhClient, kubeClient, resourceMaps); err != nil {
 			return err
 		}
 	}
 	if err := upgradeutil.UpdateResources(namespace, lhClient, resourceMaps); err != nil {
 		return err
 	}
 
-	// When lhVersionBeforeUpgrade < v1.5.0, it is v1.4.x. The `CheckUpgradePathSupported` method would have failed us out earlier if it was not v1.4.x.
+	// When lhVersionBeforeUpgrade < v1.6.0, it is v1.5.x. The `CheckUpgradePathSupported` method would have failed us out earlier if it was not v1.5.x.
 	resourceMaps = map[string]interface{}{}
 	if semver.Compare(lhVersionBeforeUpgrade, "v1.5.0") < 0 {
-		logrus.Info("Walking through the resource status upgrade path v1.4.x to v1.5.0")
-		if err := v14xto150.UpgradeResourcesStatus(namespace, lhClient, kubeClient, resourceMaps); err != nil {
+		logrus.Info("Walking through the resource status upgrade path v1.5.x to v1.6.0")
+		if err := v15xto160.UpgradeResourcesStatus(namespace, lhClient, kubeClient, resourceMaps); err != nil {
```

Comment:
Should it be `if semver.Compare(lhVersionBeforeUpgrade, "v1.6.0") < 0` instead?

Reply:
Good catch!